Sparkle/SPUUpdater.m

```diff
@@ -602,9 +606,8 @@ - (void)resetUpdateCycle
         return; // not even ready yet
     }
     
-    [self cancelNextUpdateCycle];
-    
     if (self.canCheckForUpdates) {
+        [self cancelNextUpdateCycle];
         [self scheduleNextUpdateCheck];
```

Comment:
Maybe scheduleNextUpdateCheck should cancel itself?

Reply:
Maybe. They're not always called together currently so not refactoring that at the moment.